### PR TITLE
Add network route

### PR DESCRIPTION
**Pull request by yaocw2020**

### CODE REVIEW

File: docs/networking/harvester-network.md [hunk 1]
```diff
@@ -45,15 +45,16 @@ The below diagram illustrates how the VLAN network works in Harvester.
 
 ### Enabling Default VLAN Network
 
-Users can enable the VLAN network via **Setting > VLAN** and selecting a common physical NIC for the nodes as the default VLAN config .
+Users can enable the VLAN network via **Setting > VLAN** and selecting a network interface for the nodes as the default VLAN config .
```
Comment (nunix) [inline docs/networking/harvester-network.md:48]:
```suggestion
You can enable VLAN network via **Setting > VLAN**. Once enabled, you will be able to select a network interface for the nodes as the default VLAN config.
```

File: docs/networking/harvester-network.md [hunk 2]
```diff
@@ -45,15 +45,16 @@ The below diagram illustrates how the VLAN network works in Harvester.
...
 
-It is recommended to choose a separate NIC for the VLAN other than the one used for the management network (i.e., `harvester-mgmt`) for better network performance and isolation.
+It is recommended to choose a separate network interface for the VLAN other than the one used for the management network (i.e., `harvester-mgmt`) for better network performance and isolation.
```
Comment (nunix) [inline docs/networking/harvester-network.md:50]:
```suggestion
For better network performances and isolation, we recommend to choose a separate network interface for the VLAN other than the one used for the management network (i.e., `harvester-mgmt`).
```

File: docs/networking/harvester-network.md [hunk 3]
```diff
@@ -64,8 +65,13 @@ Optional: Users can customize each node's VLAN network via the **HOST > Network*
 A new VLAN network can be created via the **Advanced > Networks** page and clicking the **Create** button.
 
  - Specify the name and VLAN ID that you want to create for the VLAN network.
-
-  ![create-vlan-network.png](./assets/create-network.png)
+   ![create-vlan-network.png](assets/create-network.png)
+  
+ - Configure the route so that users can connect the IPv4 address in the VLAN network from the hosts. The CIDR and gateway of the VLAN network are necessary parameters of the route. There are two methods to get them.
```
Comment (nunix) [inline docs/networking/harvester-network.md:70]:
```suggestion
 - Configure a route in order to allow the hosts to connect to the VLAN network using IPv4 addresses. The CIDR and gateway of the VLAN network are mandatory parameters for the route configuration.  You can configure the route by choosing one of the following options:
```

File: docs/networking/harvester-network.md [hunk 4]
```diff
@@ -64,8 +65,13 @@ Optional: Users can customize each node's VLAN network via the **HOST > Network*
...
+   - auto(DHCP) mode: Harvester network controller will get the CIDR and gateway through DHCP protocol if there is a DHCP server. Moreover, users can specify the DHCP server address.
```
Comment (nunix) [inline docs/networking/harvester-network.md:71]:
```suggestion
   - auto(DHCP) mode: the Harvester network controller will get the CIDR and gateway values from the DHCP server using the DHCP protocol. You need to specify the DHCP server address.
```

File: docs/networking/harvester-network.md [hunk 5]
```diff
@@ -64,8 +65,13 @@ Optional: Users can customize each node's VLAN network via the **HOST > Network*
...
+   ![](assets/create-network-auto.png)
+   - manual mode: Users should provide CIDR and gateway manually.
```
Comment (nunix) [inline docs/networking/harvester-network.md:73]:
```suggestion
   - manual mode: You need to specify the CIDR and gateway values manually.
```

**Review by nunix — COMMENTED** [review]
Rephrased `Users` references to active voice and rephrased the VLAN configuration description and options

File: docs/networking/harvester-network.md (hunk 4 above)
Comment (yaocw2020) [reply docs/networking/harvester-network.md:71]: It's optional to specify the DHCP server address.

Comment (nunix) [reply docs/networking/harvester-network.md:71]: rephrased based on the feedback

**Review by nunix — COMMENTED** [review]
rephrased the DHCP server option

**Review by nunix — APPROVED** [review]

**Comment by guangbochen** [comment]
can we squash the commits, thanks

**Pull request merged**